.spelling

```diff
@@ -63,12 +68,15 @@ e.g.
 E10s
 ElasticSearch
 EMR
+entrypoint
+entrypoints
 ETL
 Fenix
 Fennec
 filename
 FirefoxReality
 FireTV
+firstrun
```

Comment:
This appears to be a technical term. Can you wrap the calls in backticks instead of adding an exception?